lib/code_fund/queries/user_impression.ex

```diff
@@ -0,0 +1,70 @@
+defmodule CodeFund.Query.UserImpression do
```

Comment:
Does any of this need testing or are we satisfied with the tests in the Stats module?

Reply:
I'd like to get some tests on it.